components/messages/MessageCard.tsx

```
@@ -0,0 +1,68 @@
+            source={{
+              uri: group.image || PLACEHOLDER_IMAGE,
+            }}
+            style={styles.avatarImage}
+            size={54}
+          />
+          <View style={styles.contentContainer}>
+            <View style={styles.textGroup}>
+              <Text style={styles.titleText}>
+                {group.name}
+              </Text>
+              {group?.latestMessage && group?.latestMessage?.message && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)